.github/workflows/validation.yaml

```diff
@@ -0,0 +1,36 @@
+name: Kubernetes Validation
+
+on:
+  pull_request:
+
+
+env:
+  TUTOR_ROOT: ./.github/testing
+
+jobs:
+  validation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
+      - name: Install python reqs
+        run: |
+          pip install -r ./.github/testing/requirements.txt
+          pip install -e .
+          tutor plugins list
+      - name: Tutor config save
+        run: tutor config save
```

Comment:
maybe is better to export TUTOR_ROOT and TUTOR_PLUGINS_ROOT before to avoid any kind of issue?